torch/quantization/ns/utils.py

```diff
@@ -354,6 +354,42 @@ def rekey_logger_info_on_node_name_of_model(
     return new_results
 
 
+def maybe_add_missing_fqns(results: NSResultsType) -> None:
+    """
+    If `fqn` entries are filled in for one of the models in `results`, copies
+    them over to any models which do not have them filled out.
+
+    A common use case benefitting from this is comparing a model prepared by
+    quantization to a quantized model. In this case, the model prepared by
+    quantization would have `fqn` entries, and the quantized model would not.
```

Comment:
Wondering why quantized model wouldn't have 'fqn' entries?

Reply:
I think maybe we can add node_name_to_scope to quantized model as well if that's needed, we just don't have a clear use case yet